### PR TITLE
Use complete list of DOM event handlers for detecting JS attributes

### PR DESCRIPTION
**Pull request by rhysd**
I found complete list of DOM event handler attributes in HTML spec.

https://html.spec.whatwg.org/#event-handlers-on-elements,-document-objects,-and-window-objects

I applied the list to `JS_DOM_EVENT_ATTRS` constant to detect all JS event handler attrbutes.

### CODE REVIEW

**Comment by snshn** [comment]
That's wild, had no idea about some of them in the list! Thank you :pray: 
There seems to be something rustfmt doesn't like within stable, I think if you put `// ---...----` or just `//` there instead of empty lines it will pass the checks, with us changing it back to empty lines in a couple of months when everybody is using the latest Rust where rustfmt allows them.

**Comment by rhysd** [comment]
Ahh, I'm sorry that I had installed nightly `rustfmt` to investigate some rustfmt issue previously. I reinstalled stable again and applied `cargo fmt`. I squashed the formatter changes to one commit 26e89ae.

File: src/js.rs [hunk 1]
```diff
@@ -1,29 +1,97 @@
...
+    "onoffline",
+    "ononline",
+    "onpagehide",
+    "onpageshow",
+    "onpopstate",
+    "onrejectionhandled",
+    "onstorage",
+    "onunhandledrejection",
+    "onunload",
+    // html element
```
[the remark this answers is not in the document]
Comment (rhysd) [reply src/js.rs]: Sure. I'll add better comments.

[the remark this answers is not in the document]
Comment (snshn) [reply src/js.rs]: Oh, your comments are great

**Pull request merged**